ts_src/lib/utils.ts

```diff
@@ -9,6 +10,12 @@ import {
   PsbtOutputUpdate,
 } from './interfaces';
 
+export interface ProprietaryKey {
```

Comment:
```suggestion
/**
 * Key (as in Key-Value pair)
 * https://github.com/bitcoin/bips/blob/master/bip-0174.mediawiki#proprietary-use-type
 */
export interface ProprietaryKey {
```

The file `utils.ts` is a bit generic - I believe it deserves to live in a `proprietaryData.ts`

Reply:
Done